app/views/coronavirus_landing_page/components/landing_page/_statistics_section.html.erb

```diff
@@ -7,58 +7,42 @@
   } %>
 
   <% if statistics && statistics.cumulative_vaccinations %>
-    <div>
-      <%= render "govuk_publishing_components/components/heading", {
-        text: "Total vaccinations"
-      } %>
-
-      <span class="govuk-heading-xl">
-        <%= number_with_precision(statistics.cumulative_vaccinations / 1_000_000.0, precision: 1) %>
-        million
-      </span>
-
-      <p class="govuk-body">people have received their first dose</p>
-
-      <p class="govuk-body">
-        Last updated <%= statistics.cumulative_vaccinations_date.strftime("%d %B %Y") %>
-      </p>
-    </div>
+    <%= render "coronavirus_landing_page/components/landing_page/statistic_section_block", {
```

Comment:
You can probably simplify this name to `coronavirus_landing_page/components/landing_page/statistic` and that'd match better with the class name structure you're using.